### PR TITLE
Translate accessibility-statement

### PR DESCRIPTION
**Pull request by mindsers**

### CODE REVIEW

File: docs/accessibility-statement.md [hunk 1]
```diff
@@ -1,48 +1,48 @@
 ---
-title: Accessibility Statement
+title: Déclaration d'accessibilité
 ---
 
-_Last updated: April 8, 2019_
+_Mis à jour le: 8 avril 2019_
 
-We want everyone who uses Gatsby to feel welcome and find the experience rewarding: this includes developers with disabilities and users with disabilities for Gatsby sites, docs, and resources.
+Nous souhaitons que tous ceux qui utilisent Gatsy se sentent bienvenue et trouve l'expérience gratifiante: ceci inclu les développeurs ayant des handicaps et les utilisateurs ayant des handicaps lorsqu'il s'agit des sites Gatsby, de la documentation, et des autres ressources.
```
Comment (SamuelAlev) [inline docs/accessibility-statement.md]:
```suggestion
Nous souhaitons que tous ceux qui utilisent Gatsy se sentent bienvenue et trouve l'expérience gratifiante: ceci inclut les développeurs ayant des handicaps et les utilisateurs ayant des handicaps lorsqu'il s'agit des sites Gatsby, de la documentation, et des autres ressources.
```

File: docs/accessibility-statement.md [hunk 2]
```diff
@@ -1,48 +1,48 @@
...
 
-This page was created to collect accessibility information about the Gatsby ecosystem in one place and provide communication channels for people with disabilities to get help with Gatsby.
+Cette page a été créé pour ressembler en un seul endroit les informations sur l'accessibilité dans l'éco-système Gatsby et indique les canals de communication où les personnes avec des handicaps peuvent trouver de l'aide concernant Gatsby.
```
Comment (SamuelAlev) [inline docs/accessibility-statement.md]:
```suggestion
Cette page a été créée pour ressembler en un seul endroit les informations sur l'accessibilité dans l'éco-système Gatsby et indique les canaux de communication où les personnes avec des handicaps peuvent trouver de l'aide concernant Gatsby.
```

File: docs/accessibility-statement.md [hunk 3]
```diff
@@ -1,48 +1,48 @@
...
 
-## Gatsby products and services
+## Les produits et services Gatsby
 
-Gatsbyjs.org is the online home of the open source Gatsby website framework; it includes documentation, tutorials, and guides on how and why to build with Gatsby, as well as a blog, site showcase, and information on how to contribute to the project.
+Gatsbyjs.org est le site internet du framework open-source Gatsby; cela inclu la documentation, les tutoriels, et les guides traitant du comment et du pourquoi construire avec Gatsby, ainsi qu'un blog, un espace de demonstration et des information sur comment contribuer au projet.
```
Comment (SamuelAlev) [inline docs/accessibility-statement.md]:
```suggestion
Gatsbyjs.org est le site internet du framework open-source Gatsby; cela inclut la documentation, les tutoriels, et les guides traitant du comment et du pourquoi construire avec Gatsby, ainsi qu'un blog, un espace de démonstration et des informations sur comment contribuer au projet.
```

File: docs/accessibility-statement.md [hunk 4]
```diff
@@ -1,48 +1,48 @@
...
 
-Gatsbyjs.com is the business website for Gatsby, Inc. the startup building Gatsby, including our first product, Gatsby Preview: a cloud service allowing team members to collaborate on a Gatsby site in development.
+Gatsbyjs.com est le site internet commercial de Gatsby, Inc. la startup qui développe Gatsby, cela inclu notre premier produit, Gatsby Preview : un service cloud qui permet aux membres d'une équipe de collaborer au développement d'un site Gatsby.
```
Comment (SamuelAlev) [inline docs/accessibility-statement.md]:
```suggestion
Gatsbyjs.com est le site internet commercial de Gatsby, Inc. la startup qui développe Gatsby, cela inclut notre premier produit, Gatsby Preview : un service cloud qui permet aux membres d'une équipe de collaborer au développement d'un site Gatsby.
```

File: docs/accessibility-statement.md [hunk 5]
```diff
@@ -1,48 +1,48 @@
...
 
-## Providing feedback and getting accessibility help
+## Partager vos retours et recevoir de l'aide concernant l'accessecibilité
```
Comment (SamuelAlev) [inline docs/accessibility-statement.md]:
```suggestion
## Partager vos retours et recevoir de l'aide concernant l'accessibilité
```

File: docs/accessibility-statement.md [hunk 6]
```diff
@@ -1,48 +1,48 @@
...
 
-We strive to make these websites and the Gatsby framework itself as accessible as possible. Our goal is to meet [WCAG 2.0 AA](https://www.w3.org/TR/WCAG20/), with which we are partially compliant. We’ll be the first to admit the Gatsby ecosystem is a work in progress, and we are open to all feedback to make things better.
+Nous nous efforçons de rendre les sites et le frameworkd Gatsy lui-même aussi accessible que possible. Notre objectif est d'atteindre la norme [WCAG 2.0 AA](https://www.w3.org/TR/WCAG20/) avec laquelle nous sommes en partie conforme. Nous serons les premier à admettre que l'éco-système Gatsby est en constant développement et nous somme ouvert à tous les retours pour rendre les choses meilleures.
```
Comment (SamuelAlev) [inline docs/accessibility-statement.md]:
```suggestion
Nous nous efforçons de rendre les sites et le framework Gatsy lui-même aussi accessible que possible. Notre objectif est d'atteindre la norme [WCAG 2.0 AA](https://www.w3.org/TR/WCAG20/) avec laquelle nous sommes en partie conforme. Nous serons les premier à admettre que l'éco-système Gatsby est en constant développement et nous sommes ouvert à tous les retours pour rendre les choses meilleures.
```

**Review by SamuelAlev — DISMISSED** [review]

**Review by dantereve — APPROVED** [review]
Looks great! Well done! 💪

**Comment by dantereve** [comment]
@SamuelAlev is it ok for you now?

**Review by dantereve — APPROVED** [review]

**Review by gatarelib — APPROVED** [review]

**Pull request merged**